client/state/purchases/selectors/get-purchases.js

```diff
@@ -10,6 +10,6 @@ import 'calypso/state/purchases/init';
  * @returns {Array} Purchases
  */
 export const getPurchases = createSelector(
-	( state ) => createPurchasesArray( state.purchases.data ),
-	( state ) => [ state.purchases.data ]
```

Comment:
Even though there's nothing wrong with it, we can probably remove this change (I think) now that your tests are providing `purchases.data`. That will simplify the diff.